app/views/servicecommissioningstatus/ServiceCommissioningStatusPage.scala.html

```diff
@@ -0,0 +1,115 @@
+ * limitations under the License.
+ *@
+
+@import uk.gov.hmrc.cataloguefrontend.servicecommissioningstatus.ServiceCommissioningStatus
+@import uk.gov.hmrc.cataloguefrontend.servicecommissioningstatus.StatusCheck
+@import uk.gov.hmrc.cataloguefrontend.model.Environment
+@import uk.gov.hmrc.cataloguefrontend.model.Environment.Integration
+
+@this()
+
+@(serviceCommissioningStatus: ServiceCommissioningStatus
+)(implicit
+  messages: Messages,
+  request: Request[_]
+)
+    @standard_layout("Service-Commissioning-Status") {
+        <header>
+            <h1>Service Commissioning Status: @serviceCommissioningStatus.serviceName</h1>
+        </header>
+
+        <div class="row">
+            <div class="col-sm-6">
+                <table class="table table-striped">
+                    <thead>
+                        <tr>
+                            <th>Status Check</th>
+                            <th colspan="6">Service Commissioning Status</th>
+                        </tr>
+                    </thead>
+                    <tbody>
+                        <tr>@simpleStatusCheck("Github Repo", serviceCommissioningStatus.hasRepo.evidence)</tr>
+
+                        <tr>@simpleStatusCheck("App Config Base", serviceCommissioningStatus.hasAppConfigBase.evidence)</tr>
+
+                        <tr>@environmentStatusCheck("App Config Environment", serviceCommissioningStatus.hasAppConfigEnv.asMap)</tr>
+
+                        <tr>@environmentStatusCheck("Frontend Routes", serviceCommissioningStatus.hasFrontendRoutes.asMap)</tr>
+
+                        <tr>@simpleStatusCheck("Build Jobs", serviceCommissioningStatus.hasBuildJobs.evidence)</tr>
+
+                        <tr>@simpleStatusCheck("Service Manager Config", serviceCommissioningStatus.hasSMConfig.evidence)</tr>
+
+                        <tr>@environmentStatusCheck("Deployed", serviceCommissioningStatus.isDeployed.asMap)</tr>
+
+                        <tr>@simpleStatusCheck("Kibana", serviceCommissioningStatus.hasDashboards.grafana.evidence)</tr>
+
+                        <tr>@simpleStatusCheck("Grafana", serviceCommissioningStatus.hasDashboards.grafana.evidence)</tr>
+
+                        <tr>@simpleStatusCheck("Alerts", serviceCommissioningStatus.hasAlerts.evidence)</tr>
+                    </tbody>
+                </table>
+            </div>
+        </div>
+    }
+
+@simpleStatusCheck(statusCheck: String, evidence: Option[String]) = {
+    <td>@statusCheck</td>
+
+    <td style="text-align: center">
+        @evidence match {
+            case Some(evidence) => {
+                <span class="glyphicon glyphicon-ok low"></span>
+                <a href="@evidence" target="_blank" rel="noopener noreferrer">[view]</a>
+            }
+            case None => {
+                <span class="glyphicon glyphicon-minus archived"></span>
+            }
+        }
+    </td>
+    @for(x <- 1 until Environment.values.length) {
```

Comment:
or `@Environments.values.map { _ =>`

Reply:
I need 5 empty <td> to make the display look good, map on Environment would be 6

Reply:
Thanks fixed